### PR TITLE
update plugin forms

### PR DESCRIPTION
**Pull request by jdalsem**

### CODE REVIEW

File: mod/reportedcontent/views/default/forms/reportedcontent/add.php [hunk 1]
```diff
@@ -12,50 +12,40 @@
...
-</div>
+foreach ($fields as $field) {
+	echo elgg_view_field($field);
+}
+
+$footer = elgg_view('input/submit', array(
+	'value' => elgg_echo('reportedcontent:report'),
+));
+$footer .= elgg_view('input/button', [
+	'class' => 'elgg-button-cancel mls',
```
Comment (hypeJunction) [inline mod/reportedcontent/views/default/forms/reportedcontent/add.php]: I don't think `mls` is needed any more.

**Review by hypeJunction — APPROVED** [review]

File: mod/reportedcontent/views/default/forms/reportedcontent/add.php (hunk 1 above)
Comment (jdalsem) [inline mod/reportedcontent/views/default/forms/reportedcontent/add.php]: it is in this case... could change it to a horizontal aligned fieldset... i believe then it is not required...

**Comment by hypeJunction** [comment]
Just add the style for `.elgg-button + .elgg-button` and be done with it.

**Comment by jdalsem** [comment]
> Just add the style for .elgg-button + .elgg-button and be done with it.

will do

**Pull request merged**